resources/hvac.rb

```diff
@@ -2137,6 +2135,28 @@ def self.apply_dehumidifier(model, runner, energy_factor, integrated_energy_fact
     zone_hvac.addToThermalZone(control_zone)
   end
 
+  def self.dehumidifier_ief_to_ef_inputs(w_coeff, ef_coeff, ief, water_removal_rate)
+    # Shift inputs under IEF test conditions to E+ supported EF test conditions
+    # test conditions
+    ief_db = UnitConversions.convert(65.0, "F", "C") # degree C
+    rh = 60.0 # for both EF and IEF test conditions, %
+
+    # Independent ariables applied to curve equations
+    var_array_ief = [1, ief_db, ief_db * ief_db, rh, rh * rh, ief_db * rh]
+
+    # Curved values under EF test conditions
+    curve_value_ef = 1 # Curves are nomalized to 1.0 under EF test conditions, 80F, 60%
+    # Curve values under IEF test conditions
+    ef_curve_value_ief = var_array_ief.zip(ef_coeff).map { |var, coeff| var * coeff }.inject(0, :+)
+    water_removal_curve_value_ief = var_array_ief.zip(w_coeff).map { |var, coeff| var * coeff }.inject(0, :+)
```

Comment:
@jonwinkler If the IEF is specified instead of EF, I am assuming the `Capacity` input is also tested under new IEF test conditions, so I shifted it as well. Is it reasonable?